File: API/Entities/Product.cs
namespace API.Entities;

public class Product
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public string PictureUrl { get; set; }
    public string PictureUrl2 { get; set; }
    public string PictureUrl3 { get; set; }
    public string Sex { get; set; }
    public string Brand { get; set; }
    public long Price { get; set; }
    public List<InventoryItem> InventoryItems { get; set; } = new ();
    public string PublicId { get; set; }
    public string PublicId2 { get; set; }
    public string PublicId3 { get; set; }

}

File: client/src/app/models/product.ts
export interface Product {
    id: number;
    name: string;
    description: string;
    pictureUrl: string;
    pictureUrl2: string;
    pictureUrl3: string;
    sex: string;
    brand: string;
    price: number;
}

export interface InventoryItem {
    id: number;
    sizeMl: number;
    pricePercent: number;
    quantityInStock: number;
}

export interface ProductParams {
    orderBy: string;
    searchTerm?: string;
    sexs: string[];
    brands: string[];
    pageNumber: number;
    pageSize: number;
    
}